### PR TITLE
feat: migration UI set opt-out options

### PR DESCRIPTION
**Pull request by elevatebart**
- Closes UNIFY-934

### User facing changelog

Adds a change button to allow users to pt out from renaming files.

To test, launch component testing in `@packages/launchpad` and run the file `RenameSpecAuto.cy.ts`

### Additional details

- `Radio.vue` now accepts an `option` scoped slot to customize the appearance of options
- A slight derivation from the given screen: simple warnings cannot have rich-text so code and bold have been removed.

### Screenshots

Screen Shot 2022-01-19 at 8 45 16 PM

Screen Shot 2022-01-19 at 8 45 37 PM

Screen Shot 2022-01-19 at 8 45 59 PM

Screen Shot 2022-01-19 at 8 46 21 PM

### PR Tasks

- [x] Have tests been added/updated?

### CODE REVIEW

File: packages/launchpad/src/migration/RenameSpecsAuto.vue [hunk 1]
```diff
@@ -68,4 +108,15 @@ fragment RenameSpecsAuto on Migration {
...
+const step1Modal = ref(false)
+const step2Modal = ref(false)
+
+// probably to be changed for a GQL field
+const skipRename = ref(false)
+
+function applySkipResult (val:string) {
+  // TODO: add a GQL mutation here
+  skipRename.value = val === 'skip'
+}
```
Comment (lmiller1990) [inline packages/launchpad/src/migration/RenameSpecsAuto.vue:121]: When is this happening? Is there a ticket to capture this?

**Review by lmiller1990 — COMMENTED** [review]
Looks fine, standard UI stuff - left one q, we have some TODOs here, assuming @ImCesar has more context and will be wiring up the graphql stuff?

**Comment by brian-mann** [comment]
Could we write the warnings/messages in markdown and then parse them with a markdown formatted? This is needed for other areas in the UI and would give you a simple syntax for expressing formatted messages

**Comment by lmiller1990** [comment]
Why markdown which we parse? We should be able to just make reusable components.

**Comment by elevatebart** [comment]
@brian-mann, this is a good idea. 
I will add this to the backlog; this should be a quick win.

@lmiller1990 I understand brian's idea. We want to write rich-text error messages in the back-end, and markdown is a perfect language for this. Then send them over the wire. The Alert.vue reusable component could render its title using markdown, which would be helpful.

**Pull request merged**